ai/fine-tuning-v1/general-wires/detach_query.shs

```diff
@@ -0,0 +1,24 @@
+{
+  prompt: """
+    How do I execute one wire that will run asynchornously to the main wire.
+  """
+
+  solution: """
+   @wire(wire{
+    Detach(wire-to-run-asynchronously)
+   })
+  """
+
+  explicit: """
+   @wire(wire{
+    Detach(Wire:wire-to-run-asynchronously)
+   })
+  """
+
+  explanation: """
+    Detach will  execute a wire concurrently in the background within the same mesh as the parent wire. Only one wire of each unique wire
+    can be detached at any point in time. Subsequent detaches of the same wire, will be ignored. 
```

Comment:
Unless restart parameter is specified, also the whole one wire of each unique wire I'm not sure